head.html

```diff
@@ -129,7 +138,44 @@
           topicsWrap.appendChild(btn);
         });
         // topicsWrap.appendChild(btnWrap);
-        div.parentNode.appendChild(topicsWrap);
+        last.parentNode.appendChild(topicsWrap);
+      }
+    }
+  }
+
+  function fetchProducts() {
+    const last = getSection();
+    const insertInside = getSection(2);
+    if (insertInside) {
+      insertInside.classList.add('left');
+      let topics, div;
+      Array.from(last.children).forEach((i) => {
+        const r = /^Products\: (.*)$/gmi.exec(i.innerText);
+        if (r && r.length > 0) {
+          products = r[1].split(',');
+          div = i;
+        }
+      });
+      if (products) {
+        div.remove();
```

Comment:
on post pages, the `div` is potentially shared between topics and products, so the first one removes the div and the second one loses...

Reply:
It works because execution is sequential: first get data for topics, second get data for products, last remove the div. But I agree I could move this in a cleanup function for clarity.

Reply:
arff.. no, it is just confusing because what is being removed is the `<p>` not the `<div>` (container of the product text, same for topics)